extensions/target-specific/chromium/win-d3d11-angletextures/index.html

```diff
@@ -0,0 +1,76 @@
+<!--
+    MIT License
+
+    Copyright (c) Microsoft Corporation. All rights reserved.
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Inspect Compositor Frame</title>
```

Comment:
nit: consider using the extension name (or something similar) for the title

Reply:
done